tests/unit_node/sqlite_test.ts

```diff
@@ -197,3 +197,23 @@ CREATE TABLE two(id int PRIMARY KEY) STRICT;`);
 
   db.close();
 });
+
+Deno.test("[node/sqlite] StatementSync#iterate", () => {
+  const db = new DatabaseSync(":memory:");
+  const stmt = db.prepare("SELECT 1 UNION ALL SELECT 2 UNION ALL SELECT 3");
+  // @ts-ignore: types are not up to date
```

Comment:
Did y'all forget to eventually add these in? 😅 

I'm able to call stmt.iterate() in Deno 2.2.3, but the types are not present so `deno check` fails.